src/react/components/pages/homepage/homePage.test.tsx

```diff
@@ -4,26 +4,37 @@ import { Provider } from "react-redux";
 import { BrowserRouter as Router, Link } from "react-router-dom";
 import { AnyAction, Store } from "redux";
 import MockFactory from "../../../../common/mockFactory";
+import { StorageProviderFactory } from "../../../../providers/storage/storageProviderFactory";
 import { IApplicationState, IProject, AppError, ErrorCode } from "../../../../models/applicationState";
 import IProjectActions, * as projectActions from "../../../../redux/actions/projectActions";
+import IApplicationActions, * as applicationActions from "../../../../redux/actions/applicationActions";
 import createReduxStore from "../../../../redux/store/store";
 import ProjectService from "../../../../services/projectService";
+import ImportService from "../../../../services/importService";
 import CondensedList from "../../common/condensedList/condensedList";
+import Confirm, { IConfirmProps } from "../../common/confirm/confirm";
 import FilePicker, { IFilePickerProps } from "../../common/filePicker/filePicker";
 import HomePage, { IHomepageProps, IHomepageState } from "./homePage";
 
 jest.mock("../../common/cloudFilePicker/cloudFilePicker");
 import { CloudFilePicker, ICloudFilePickerProps } from "../../common/cloudFilePicker/cloudFilePicker";
 
 jest.mock("../../../../services/projectService");
+jest.mock("../../../../services/importService");
```

Comment:
Does this work? I was under the impression that the import statement needed to come after the mock statement. But that'd be great if I'm wrong!

Reply:
It does work, but it's not as clean/pretty. I changed it so it goes with convention :)